cel.go

```diff
@@ -177,8 +177,13 @@ func NewActionActivation(stepName string, transaction *pbcodec.TransactionTrace,
 		"transaction_id":    transaction.Id,
 		"transaction_index": transaction.Index,
 		"step":              stepName,
-		"global_seq":        trace.Receipt.GlobalSequence,
-		"execution_index":   trace.ExecutionIndex,
+		"global_seq": func() interface{} {
```

Comment:
You should apply this pattern everywhere the Receipt is used in the `cel.go` file.
See for example the `NewActivation` method or `NewTableActivation`

Reply:
Adding check for `trace.Receipt` in more place, some is not really matter like dbops will not present in failed transaction so just add as precaution.